index.bs

```diff
@@ -254,6 +255,12 @@ Previous versions of this specification defined v0 of the AV1CodecConfigurationR
 
 The <dfn noexport>version</dfn> field indicates the version of the AV1CodecConfigurationRecordv1. The value SHALL be set to 1 for AV1CodecConfigurationRecordv1.
 
+The <dfn export>marker</dfn> field SHALL be set to 1.
+
+NOTE: The marker bit ensures that tje bit pattern of the first byte of the AV1CodecConfigurationRecordv1 cannot be mistaken for an [=OBU Header=] byte.
```

Comment:
Typo, should be "the" not "tje". Same in index.html

Reply:
Fixed. Thanks!